ts-serverless-quickstart/src/pets/fish.ts

```diff
@@ -0,0 +1 @@
+// Empty, we'll fill this in later
```

Comment:
* I'd add this to gitignore, since we intend to edit it
* minor: do we actually need this file? or can we just tell people to create it (rather than edit)?

Reply:
I wanted to add it as empty just to make sure ppl were aware that we're going to use it. They could create it, but it felt like less effort for the user to do that then have it ready for them